sdks/java/io/cassandra/build.gradle

```diff
@@ -18,11 +18,23 @@
 
 plugins { id 'org.apache.beam.module' }
 // Do not relocate guava to avoid issues with Cassandra's version.
-applyJavaNature(shadowClosure: DEFAULT_SHADOW_CLOSURE << {
-  dependencies {
-    exclude(dependency(project.library.java.guava))
-  }
-})
+applyJavaNature(
+    shadowClosure: {
+      dependencies {
+        // is default, but when omitted the default action is to include all runtime deps
+        include(dependency(project.library.java.guava))
+
+        // hack: now exclude the only thing that was included
+        exclude(dependency(project.library.java.guava))
+
```

Comment:
The default is the `runtime` configuration and it has to be overridden. We never did that because we use the dependency filters.

Something that I think should work instead is this:

```groovy
shadowJar {
  configurations = [project.configurations.compile]
}
```

I tried the above and it did not work. I think maybe it is the order of the concatenation of the `shadowClosure` with the parts of the configuration that are hardcoded into the main plugin. I am not sure. And I'm not sure if there is a var I can just set `dependencies = []`. Basically we want to turn off the shadow plugin here but I don't want to make such a big change.

I manually confirmed the contents of the jar.